tests/test_convert.py

```diff
@@ -0,0 +1,746 @@
+# Copyright 2023 Dgraph Labs, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests conversion functions."""
+
+import unittest
+import json
+
+from pydgraph import convert
+
+class TestConvert(unittest.TestCase):
+    """Tests conversion functions."""
+
+    def test_extract_nodes_edges(self):
```

Comment:
Wouldn't it be better if we could use Dgraph responses and apply the extract_nodes function instead? What if the response changes tomorrow but we forget to improve/modify this function?

Reply:
If I understand you correctly, you are describing an integration test, whereas this is a simple unit test.  If the response changes tomorrow (unlikely, but maybe months later) we will see a failure here and fix it.  Writing an integration test here might be a good exercise for one of our junior engineers.  :smile_cat:

Reply:
Will you write one?

Reply:
Open an issue and tackle it there in github? We can tag it as a `good-first-issue` or something similar. I don't see the reason to block this (well it's already merged so...). This is a nifty utility to integrate with graphistry etc.

Reply:
I can open an issue and think of who to give it to.  Apologies if that came off as a bit acerbic - I literally meant that it is a nice exercise and someone can learn something from doing it.